src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -100,6 +100,7 @@
         </li>
         @if (Model.ShellSettingsEntries.Count > 0)
         {
+
```

Comment:
remove extra line

Reply:
I remove it before I saw your comment ;)